cpp/examples/luma.gl/animometer.cc

```diff
@@ -131,7 +131,8 @@ auto createAttributeTable(wgpu::Device device) -> std::shared_ptr<garrow::Table>
 }
 
 int main(int argc, const char* argv[]) {
-  GLFWAnimationLoop animationLoop;
+  GLFWAnimationLoop::Options options;
+  GLFWAnimationLoop animationLoop{options};
```

Comment:
Nit: If options can be empty maybe we should not need to provide them here. I guess it does show that you can provide options...